prometheus/hooks/run.orig

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+exec 2>&1
+
+<<<<<<< HEAD
```

Comment:
conflicts but also `run.orig` is not a file you are intending to check in right?